packages/helper/src/email/index.ts

```diff
@@ -157,7 +158,7 @@ const sendVerificationEmail = async (
 		.replace('%buttonText%', 'Confirm Your Email');
 
 	const newEmail: IEmail = {
-		from: process.env.DEFAULT_EMAIL,
+		from: process.env.DEFAULT_EMAIL!,
```

Comment:
@Jacobjeevan for process variables you can use ```!``` for non-null coearcion since those variables are always filled